drl_world:
    version: 4
    objects:
        - name                : object_robocup_box
          number              : 5
          mirrored_object     : false
          initPose            :
              positionX     : 3..15
              positionY     : 0
              positionZ     : -1.2..0
              orientationR  : 0.0..0.23
              orientationP  : -0.23..0.23
              orientationY  : -3.141..3.141

        - name                : object_robocup_wall250
          number              : 5
          mirrored_object     : true
          shift               : 1.21
          initPose            :
              positionX     : 3..15
              positionY     : 0
              positionZ     : 0
              orientationR  : 0
              orientationP  : 0
              orientationY  : -3.141..3.141



